src/cascadia/TerminalApp/Jumplist.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+// Module Name:
+// - Jumplist.h
+//
+// Abstract:
+// - The Jumplist is the menu that pops up when right clicking a pinned
+// item in the taskbar. This class handles updating the Terminal's jumplist
+// using the Terminal's settings.
+//
+
+#pragma once
+
+#include "CascadiaSettings.h"
+#include "Profile.h"
+
+#include <ShObjIdl.h>

```

Comment:
if possible, leave this out of the public header for Jumplist.h and just forward-declare the types (structs?) you'll be using.